language/move-vm/integration-tests/src/tests/loader_tests.rs

```diff
@@ -129,82 +128,178 @@ impl Adapter {
     }
 }
 
-#[derive(Clone, Debug)]
-struct DataStore {
-    modules: HashMap<ModuleId, Vec<u8>>,
-}
+const MODULES: &str = r#"
```

Comment:
I had another test (when I was trying to commit the PR that did not load dependent modules in a `publish_module`) that was using the same set of modules and I think that is a useful way to do things, rather than having only a string to the API

Reply:
Yeah certainly no harm to have another option. I'll add that back and revert the changes to the loader tests.